torch/distributed/fsdp/_common_utils.py

```diff
@@ -0,0 +1,23 @@
+from enum import auto, Enum
```

Comment:
I am maintaining this `_common_utils.py` as I refactor. Eventually, we will merge `_utils.py` into `_common_utils.py` or other files.